upgrading/migrating_etcd.adoc

```diff
@@ -118,38 +118,67 @@ endif::[]
 The following procedure describes the steps required to successfully migrate the
 cluster (implemented as part of the Ansible etcd migration playbook).
 
-. +++<b>Create an etcd backup.</b>+++ See
+. Create an etcd backup. See
 xref:../admin_guide/backup_restore.adoc#cluster-backup[Backup and Restore]
 for steps.
 
-. +++<b>Stop masters and wait for etcd convergence:</b>+++
+. Stop masters and wait for etcd convergence:
 
 .. Stop all master services:
 +
+ifdef::openshift-enterprise[]
 ----
-# systemctl stop atomic-openshift-master-api atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master-api origin-master-controllers
+----
+endif::[]+
 
 .. Before the migration can proceed, the etcd cluster must be healthy
 and raft indices of all etcd members must differ by one unit at most.
 At the same time, all etcd members and master daemons must be stopped.
 +
-To check the etcd cluster is healthy you can run:
+To check the etcd cluster is healthy run the following:
++
+---- 
+# etcdctl --ca-file=/etc/etcd/master.etcd-ca.crt  \
+--cert-file=/etc/etcd/master.etcd-client.crt \
+--key-file=/etc/etcd/master.etcd-client.key \
+--endpoints https://etcd-test-1:2379,https://etcd-test-2:2379,https://etcd-test-3:2379 \
+cluster-health
+----
++
+The output is similar to the following for a healthy cluster:
 +
 ----
-# etcdctl <certificate_details> <endpoint> cluster-health <1>
 member 2a3d833935d9d076 is healthy: got healthy result from https://etcd-test-1:2379
 member a83a3258059fee18 is healthy: got healthy result from https://etcd-test-2:2379
 member 22a9f2ddf18fee5f is healthy: got healthy result from https://etcd-test-3:2379
 cluster is healthy
 ----
-<1> For `<certificate_details>`, see
-xref:../admin_guide/backup_restore.adoc#adding-addtl-etcd-members[Backup and Restore] for an example of how to set certificate flags.
 +
-To check a difference of raft indices you can run:
+To check a difference of raft indices run the following:
++
+----
+# ETCDCTL_API=3 etcdctl --cacert="/etc/etcd/master.etcd-ca.crt" \
```

Comment:
The original bug says that you need to run `# export ETCDCTL_API=3` before you run this command. Scott said to remove it before you run cluster-health, but IDK if you really need to fun it before you check the raft indices.

Reply:
@kalexand-rh It appears to me that the `export` command is needed based on Scott's comment. https://github.com/openshift/openshift-docs/pull/9189#discussion_r189614367

@sdodson Can you confirm?

Reply:
`export FOO=bar` sets FOO for all subsequent commands until it's unset. Where as setting, without export, in each command ensures that it's only used for that command. This document interleaves a lot of v2 and v3 commands so it's best to just set it on the specific commands that are v3. Everything looks fine as is right now.